test/index.spec.js

```diff
@@ -121,6 +129,19 @@ describe('ipns', function () {
     expect(entryDataCreated.validityType).to.equal(unmarshalledData.validityType)
     expect(entryDataCreated.signature).to.equalBytes(unmarshalledData.signature)
     expect(entryDataCreated.sequence).to.equal(unmarshalledData.sequence)
+    expect(entryDataCreated.ttl).to.equal(unmarshalledData.ttl)
+
+    if (!unmarshalledData.signatureV2) {
+      throw new Error('No v2 sig found')
```

Comment:
nit: can we expect to have property instead?

Reply:
That would certainly be my preference!  The problem here is that the type of `.signatureV2` is `Uint8Array | undefined` and the type of `arg` in `.equalsBytes(arg)` is `ArrayLike<number> | string` and the two are incompatible since `arg` cannot be `undefined`.

By checking if `.signatureV2` has a truthy value `tsc` knows it's `Uint8Array` which is then compatible with `ArrayLike<number>`.

Reply:
https://github.com/slowli/chai-bytes/issues/23